File: abc/abc165/abc165_a.cs
// https://atcoder.jp/contests/abc165/submissions/12649774

using System;
using System.Linq;

class Program
{
    static void Main()
    {
        var K = int.Parse(Console.ReadLine());
        var input = Console.ReadLine().Split().Select(int.Parse).ToArray();
        var A = input[0];
        var B = input[1];

        if (A % K == 0 || A + (K - A % K) <= B)
        {
            Console.WriteLine("OK");
        }
        else
        {
            Console.WriteLine("NG");
        }
    }
}

File: abc/abc165/abc165_a.fs
// https://atcoder.jp/contests/abc165/submissions/12651140

let K = stdin.ReadLine() |> int
let [| A; B |] = stdin.ReadLine().Split() |> Array.map int

if A % K = 0 || A + (K - A % K) <= B then "OK"
else "NG"
|> stdout.WriteLine
